src/dev/flang/parser/Lexer.java

```diff
@@ -2518,7 +2518,14 @@ else if (
                     }
                   if (c < 0)
                     {
-                      Errors.unknownEscapedChar(sourcePos(), p, escapeChars);
+                      if (p == (int) '\n')
+                        {
+                          // codepoint is skipped
+                        }
+                      else
```

Comment:
Would be very nice to add `'\n'` to `escapedChars` and handle this in the loop just before this code. 
For that to work, you will need to change the type of `escapedChars` to `init[][]` such that this could be `-1`.
The reason is that chances are much lower that this escape sequence gets overlooked in documentation etc. 

My suggestion for the code, another example where Fuzion's loops with `until` and `else` would fit well:
```
                  var i = 0;
                  while (i < escapeChars.length && p != (int) escapeChars[i][0])
                    {
                      i++;
                    }
                  if (i < escapeChars.length)
                    {
                      c = (int) escapeChars[i][1];
                    }
                  else
                    {
                      Errors.unknownEscapedChar(sourcePos(), p, escapeChars);
                    }
```

Soon, I would like to start adding a Fuzion reference manual. Things like the table of escape sequenced should then be created directly from the sources using asciidoc or similar markup.